aligner/helper_scripts/scripts.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
```

Comment:
Again, should be a function with the signature:

def shuffle_scp(data, limit=None)

Where it returns the first X if limit=X, otherwise return the full shuffled list if limit=None